dev-env

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -eu
+
+declare -a POSSIBLE_DOCKER_SOCK_PATHS=(
+    /run/docker.sock
+    /var/run/docker.sock
+)
+DOCKER_SOCK_PATH=
+readonly DOCKER_GROUP_ID=$(getent group docker | cut -d: -f3)
```

Comment:
We need GID of a user able to run Docker, so we can pass it to the container. This is because docker commands run in container will still be run via Docker socket so user running this container needs to be able to work with it.

Reply:
I'll add it above the `docker run` command instead.